internal/api/v1beta1connect/user.go

```diff
@@ -84,6 +85,8 @@ func (h *ConnectHandler) GetCurrentAdminUser(ctx context.Context, request *conne
 
 func (h *ConnectHandler) CreateUser(ctx context.Context, request *connect.Request[frontierv1beta1.CreateUserRequest]) (*connect.Response[frontierv1beta1.CreateUserResponse], error) {
 	logger := grpczap.Extract(ctx)
+	errorLogger := NewErrorLogger()
```

Comment:
can we pass the function name and the request during initialization?
eg `errorLogger := NewErrorLogger().WithRequest(request).WithFunctionName('CreateUser')`